db/migrations/20200204-drop-encrypted_key.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE job_keys DROP COLUMN encrypted_key;
```

Comment:
@dhgreene Is this going to affect anything in SSAS?  I doubt it, but just making sure...
@tbellj I updated BCDA-2681 (deployment of `r46`) to indicate that this DB Migration needs to occur.